qt/python/mantidqtinterfaces/mantidqtinterfaces/dns_single_crystal_elastic/scripts/md_single_crystal_elastic.py

```diff
@@ -0,0 +1,63 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2023 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+"""
+DNS script helpers for reduction of single crystal elastic data.
+"""
+
+from mantid.simpleapi import BinMD, LoadDNSSCD, mtd
+
+
+def load_all(data_dict, binning, params, standard=False):
+    """
+    Loading of multiple DNS files given in a dictionary to workspaces.
+    """
+    workspace_names = {}
+    for sample_name, fields in data_dict.items():
+        workspace_names[sample_name] = []
+        path = data_dict[sample_name]["path"]
+        for field_name, file_numbers in fields.items():
+            if field_name != "path":
+                workspace_name = "_".join((sample_name, field_name))
+                workspace_names[sample_name].append(workspace_name)
+                load_binned(workspace_name, binning, params, path, file_numbers, standard)
+    return workspace_names
+
+
+def load_binned(workspace_name, binning, params, path, file_numbers, standard):
+    """
+    Loading of multiple DNS datafiles into a single workspace.
+    """
```

Comment:
File names seem to follow a specific pattern for DNS. Would be useful to document it as a comment, with a sample file name.